index.html

```diff
@@ -161,6 +163,42 @@ <h3 class="lang"><a href="#">Skills</a><i class="fas fa-angle-right"></i></h3>
 				<hr>
 			</section>
 		</section>
+
+
+
+
+
+
+
+
+
+		<section class="contact-form">
+			<h2>Contact me</h2>
+			<p>If you have an application you are interested in developing, a feature that you need built or a project
+				that
+				needs coding. I’d love to help with it </p>
+			<form action="https://formspree.io/f/myyljgwb" method="POST">
+				<fieldset>
+					<ul>
+						<li>
+								<input type="text" name="name" placeholder="Elyan" maxlength="30" required>
+						</li>
+						<li>
+								<input type="email" name="email" placeholder="Johndoe@mail.com" required>
+						</li>
+						<li>
+								<textarea name="message" placeholder="write your message here" maxlength="500"></textarea>
+						</li>
+						<li class="form-button">
+							<button type="submit">Get in touch</button>
+						</li>
+					</ul>
```

Comment:
- [ ] This is not actually a UL element but a form element,[ check this link](https://www.w3schools.com/html/html_form_elements.asp) to implement the correct semantic

Reply:
Fixed. thanks